bluesky/tests/test_scan_id_source.py

```diff
@@ -0,0 +1,25 @@
+import bluesky.plans as bp
+
+
+def test_relative_pseudo(hw, RE, RE_no_scan_id, db):
```

Comment:
It might be worth giving this a more descriptive name. (Looks like it was copy/pasted from a test with a different goal that this one.)

Reply:
Good catch! Indeed, I copied over the test from another module and forgot to change the name.

Reply:
Updated to be `test_scan_id_source` via https://github.com/NSLS-II/bluesky/pull/1175/commits/7dddd6f38dd656219ac5f294274131f8c9481d6e.